<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplyFor</class>
 <widget class="QWidget" name="ApplyFor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="LabelUserName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>用户：</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPub">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>输入公钥</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelPri">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>190</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>输入私钥</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCreateDoubleKey">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>470</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>一键生成密钥对</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCreateCertificate">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>470</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>生成证书</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonSave">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>470</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存密钥对</string>
   </property>
  </widget>
  <widget class="QLabel" name="LabelCertificate">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>540</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>数字证书</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="TextEditPublicKey">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="TextEditPrivateKey">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>240</y>
     <width>261</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="TextEditCertificate">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>170</y>
     <width>301</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonBackSearch">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>30</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回查找页面</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
